<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddCharacterDialog</class>
 <widget class="QDialog" name="AddCharacterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add New Character</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Character Information</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Name:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="nameEdit"/>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="nameErrorLabel">
        <property name="visible">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">color: red; font-size: 10px;</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Surname:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="surnameEdit"/>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="surnameErrorLabel">
        <property name="visible">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">color: red; font-size: 10px;</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Age:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QSpinBox" name="ageSpin">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>150</number>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Bio:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QPlainTextEdit" name="bioEdit"/>
      </item>
      <item row="6" column="1">
       <widget class="QLabel" name="imageErrorLabel">
        <property name="visible">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">color: red; font-size: 10px;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddCharacterDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddCharacterDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
